docs/examples/DataSet/Deeper dive on qcodes doNd functionalities with a focus on arrayed sweeping benefits.ipynb

```diff
@@ -0,0 +1,455 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "dataset4 = load_by_run_spec(experiment_name='simple_sweep_do1d',\\\n",
+    "                           captured_run_id=47)\n",
+    "plot_dataset(dataset4)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.9"
+  },
+  "nbsphinx": {
+   "execute": "never"
```

Comment:
Why is execution disabled?

Reply:
cause it's slow :) in one of the comments i'm wondering if it can be sped up to always execute

Reply:
That's right. It takes nearly 10 minutes to run the notebook.

Reply:
As discussed above we should probably fix the long execution instead. 

In the future please document in the git log message why execution is disabled